clang/test/AST/Interp/literals.cpp

```diff
@@ -26,6 +28,66 @@ static_assert(number != 10, ""); // expected-error{{failed}} \
                                  // expected-note{{evaluates to}} \
                                  // ref-note{{evaluates to}}
 
+
+namespace i128 {
```

Comment:
@tbaederr 
These tests fail on targets that do not have __int128 support.
You can reporoduce the issue by adding -triple=avr to the RUN lines.

Reply:
Oof, sorry. Everything was fine this morning and then I left the house... Will fix immediately.